rust/typeql_lang.rs

```diff
@@ -20,17 +20,15 @@
  *
  */
 
-#![allow(dead_code)]
```

Comment:
There should be no more unused private functions now. (There are, but they are all the would-be-red tests).